aws-iot-accountauditconfiguration/aws-iot-accountauditconfiguration.json

```diff
@@ -0,0 +1,114 @@
+{
+  "typeName": "AWS::IoT::AccountAuditConfiguration",
+  "description": "Configures the Device Defender audit settings for this account. Settings include how audit notifications are sent and which audit checks are enabled or disabled.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-iot.git",
+  "definitions": {
+    "AuditCheckConfiguration": {
+      "description": "The configuration for a specific audit check.",
+      "type": "object",
+      "properties": {
+        "Enabled": {
+          "description": "True if the check is enabled.",
+          "type": "boolean"
+        }
+      }
+    },
+    "AuditNotificationTarget": {
+      "type": "object",
+      "properties": {
+        "TargetArn": {
+          "description": "The ARN of the target (SNS topic) to which audit notifications are sent.",
+          "type": "string",
+          "maxLength": 2048
+        },
+        "RoleArn": {
+          "description": "The ARN of the role that grants permission to send notifications to the target.",
+          "type": "string",
+          "minLength": 20,
+          "maxLength": 2048
+        },
+        "Enabled": {
+          "description": "True if notifications to the target are enabled.",
+          "type": "boolean"
+        }
+      }
+    }
+  },
+  "properties": {
+    "AccountId": {
```

Comment:
Where do we specify that this field would determine whether this resource in unique in CFN universe or not?

Reply:
Below, there's:
```
"primaryIdentifier": [
  "/properties/AccountId" 
]
```

Verified with a manual test that CFN ensures the config is unique in its universe. First created one stack, then tried to create another one and CFN threw "<accountId> already exists in stack <stack1-arn>"

Reply:
nice